contact.html

```diff
@@ -19,6 +19,9 @@
 			<li>
 			  <a href="mailto: commits@kafka.apache.org">commits@kafka.apache.org</a>: A list to track Kafka commits. To subscribe, send an email to <a href="mailto: commits-subscribe@kafka.apache.org">commits-subscribe@kafka.apache.org</a>. Archives available <a href="http://mail-archives.apache.org/mod_mbox/kafka-commits">here</a>.
 			</li>
+			<li>
+				<a href="mailto: jira@kafka.apache.org">jira@kafka.apache.org</a>: A list to track Kafka <a href="https://issues.apache.org/jira/projects/KAFKA">JIRA</a> notifications. To subscribe, send an email to <a href="mailto: jira-subscribe@kafka.apache.org">jira-subscribe@kafka.apache.org</a>. Archives available <a href="http://mail-archives.apache.org/mod_mbox/kafka-jira">here</a>.
```

Comment:
I would put this one above `commits@` since I'd expect all developers to subscribe to the JIRA one whereas I don't think there's much value in subscribing to the commits mailing list.

Also nit: indent seems inconsistent with the other ones.